src/Api/Data/DbTask.cs

```diff
@@ -6,19 +6,28 @@
 using Microsoft.Extensions.DependencyInjection;
 using Api.Models.Entity;
 using System;
+using Microsoft.AspNetCore.Hosting;
 
 namespace Api.Data
 {
     public static class DbTask
```

Comment:
I will again try my level best to define an own start up class but it was not working for me. I will try that once again.